src/main/scala/cromwell/engine/backend/jes/Pipeline.scala

```diff
@@ -26,31 +34,35 @@ object Pipeline extends LazyLogging {
 
     cpr.setParameters(jesParameters.map(_.toGoogleParameter).toVector.asJava)
 
-    logger.info(s"$tag Pipeline parameters are:\n${cpr.getParameters.asScala.map(s=>s"  $s").mkString("\n")}")
-    val pipelineId = jesConnection.genomics.pipelines().create(cpr).execute().getPipelineId
+    def createPipeline = jesConnection.genomics.pipelines().create(cpr).execute().getPipelineId
```

Comment:
This Pipeline API supports create and run, but I didn't see a way of deleting pipelines.  Given the way Cromwell uses this API, are we just forever accumulating pipelines or are they being cleaned up automatically somehow?